models/migrations/v143.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"code.gitea.io/gitea/models"
+
+	"xorm.io/xorm"
+)
+
+func recalculateStars(x *xorm.Engine) (err error) {
+	// because of issue https://github.com/go-gitea/gitea/issues/11949,
+	// recalculate Stars number for all users to fully fix it.
+
+	return models.DoctorUserStarNum()
```

Comment:
It's better to duplicate code here as there is no guarantee that model won't change in the future and that could break migration

Reply:
`gitea-vet` will also fail CI if it detects models package in a migration in the future.  
https://gitea.com/gitea/gitea-vet/pulls/5 (not released yet)